src/CarwowTheme/Filters.elm

```diff
@@ -189,7 +229,7 @@ filterView label selectedFiltersLabel filterIcon content =
                     ]
                 ]
             , div
-                [ class "tooltip-dropdown tooltip-dropdown--bottom" ]
+                [ class ("tooltip-dropdown tooltip-dropdown--" ++ alignment) ]
```

Comment:
Needed to allow the first dropdown to render in a position that will remain on-screen on small resolution desktop screens.